executor/executor.go

```diff
@@ -328,9 +330,43 @@ func (executor *Executor) execute(
 			}
 			return response, nil
 		}
-		msg := "decision executed successfully"
 
-		executor.logger.Infof(ctx, msg)
+		// short pooling to trigger pod status with max 15 minutes
+
+		statusMap := make(map[string]string)
+		statusMap["Running"] = "pod restarted successfully"
+		statusMap["Failed"] = "pod failed to restart"
+		statusMap["Unknown"] = "pod status is unknown"
+		statusMap["PodInitializing"] = "pod restarting"
+
+		backoff := 15 * time.Minute
+		msg := "pod restarting exceeded timout (15 min)"
+		start := time.Now()
+		timeout := int(backoff.Seconds())
+
+		for time.Now().Second() - start.Second() < timeout {
+
+			status := "Pending"
+
+			time.Sleep(15 * time.Second)
```

Comment:
this value and the backoff value should be defined as const value and declared in the head of the file also make sure of any of the decisions constants that are not used to be deleted too as well.

Reply:
ok